search_gov_crawler/scrapy_scheduler.py

```diff
@@ -66,10 +68,25 @@ def run_scrapy_crawl(
     )
     msg = (
         "Successfully completed scrapy crawl with args "
-        "spider=%s, allow_query_string=%s, allowed_domains=%s, start_urls=%s, output_target=%s"
+        "spider=%s, allow_query_string=%s, allowed_domains=%s, start_urls=%s, output_target=%s, search_depth=%s"
     )
     log.info(
-        msg, spider, allow_query_string, allowed_domains, start_urls, output_target
+        msg,
+        spider,
+        allow_query_string,
+        allowed_domains,
+        start_urls,
+        output_target,
+        search_depth,
+    )
+    log.info(
```

Comment:
@IsabelLaurenceau Looks like you have an extra/duplicated `log.info(...)` function